docs/sphinx/user_guide/feature/view.rst

```diff
@@ -182,7 +176,7 @@ array entry.
 
 The arguments to the ``RAJA::make_offset_layout`` method are C++ arrays that
 hold the start and end values of the indices. RAJA offset layouts support
-any number of dimensions; for example::
+any number of dimensions also; for example::

```

Comment:
Maybe "RAJA offset layouts _also_ support any number of dimensions;" ?

Reply:
Fixed